.env

```diff
@@ -22,4 +22,4 @@ LOGO_URL=''
 LOGO_WHITE_URL=''
 LOGO_TRADEMARK_URL=''
 FAVICON_URL=''
-SHOW_LEARNER_REPORT_MAINTENANCE_ALERT='true'
+SHOW_LEARNER_REPORT_MAINTENANCE_ALERT=''
```

Comment:
Should this be true to leave the banner up?

Reply:
https://github.com/edx/edx-internal/pull/3962